DogScepterLib/Core/GMData.cs

```diff
@@ -115,153 +201,160 @@ public class GMStats
         {
             MaxDegreeOfParallelism = Environment.ProcessorCount
         }
-        );
+    );
 
-        public Dictionary<string, int> StringCache;
+    //TODO: project file related?
```

Comment:
This is used as a string content -> string ID map, which can perform much better when trying to create string references (as compared to comparing every string until the correct one is located).